konfera/templates/konfera/event_speakers.html

```diff
@@ -0,0 +1,17 @@
+{% extends 'konfera/base.html' %}
+
+{% block title %}<h1>{{ event }} Speakers</h1>{% endblock %}
```

Comment:
With `<title>{% block title %}BASE TEMPLATE{% endblock %}</title>` and
`{% block title %}<h2>{{ event }} Speakers</h2>{% endblock %}` it does not display the head.

But this works `{% block title %}<title>BASE TEMPLATE</title>{% endblock %}`